### ISSUE TITLE
Cannot submit an MPI job from inside a JupyterLab terminal

### ISSUE THREAD

**Issue by ocaisa** [ComputeCanada/puppet-jupyterhub#25]
I have a simple jobscript (for EESSI):
```
#!/bin/bash -x
#SBATCH --time=00:20:00
#SBATCH --ntasks-per-node=2
#SBATCH --cpus-per-task=1
#SBATCH --nodes=2

module load GROMACS
mpirun -n 4 gmx_mpi mdrun -s ion_channel.tpr -maxh 0.50 -resethway -noconfout -nsteps 4000 -g logfile -dlb yes
```
(you can get the input file `ion_channel.tpr` [here](https://repository.prace-ri.eu/ueabs/GROMACS/1.2/GROMACS_TestCaseA.tar.gz))

This works fine if you submit from an ssh login, however if I submit from inside a JupyterLab terminal I get:
```
+ mpirun -n 4 gmx_mpi mdrun -s ion_channel.tpr -maxh 0.50 -resethway -noconfout -nsteps 4000 -g logfile -dlb yes
--------------------------------------------------------------------------
An ORTE daemon has unexpectedly failed after launch and before
communicating back to mpirun. This could be caused by a number
of factors, including an inability to create a connection back
to mpirun due to a lack of common network interfaces and/or no
route found between them. Please check network connectivity
(including firewalls and network routing requirements).
--------------------------------------------------------------------------
```

**Comment by cmd-ntrf** [comment]
Fixed in v3.8.7.

**Comment by ocaisa** [comment]
FYI, unsetting that variable was not enough for me, I had to do in order the submission to work:
```
unset SLURM_EXPORT_ENV 
unset SLURM_MEM_PER_NODE
unset SBATCH_EXPORT
```
